src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/ModuleFastPlace.kt

```diff
@@ -36,14 +37,40 @@ import java.util.function.Predicate
 object ModuleFastPlace : ClientModule("FastPlace", Category.WORLD) {
     private val cooldown by int("Cooldown", 0, 0..4, "ticks").apply { tagBy(this) }
     private val applyTo by multiEnumChoice("ApplyTo", ApplyTo.entries)
+    private val startDelay by int("StartDelay", 0, 0..1000, "ms")
```

Comment:
use ticks.
```suggestion
    private val startDelay by int("StartDelay", 0, 0..20, "ticks")
```
in code
```
startDelay * 50 // startDelay in ms
```

Reply:
but why?

Reply:
Not relevant here because we are dealing with inputs. Inputs are theoretically not on tick, so it makes sense to work with milliseconds here.